scootapi/api.go

```diff
@@ -0,0 +1,3 @@
+package scootapi
+
+// We should use go generate to run thrift --gen go:package_prefix=github.com/scootdev/scoot/scootapi/gen-go/ scoot.thrift
```

Comment:
If it doesn't function as an actual directive then I think it's a bit unusual to add this note to source code vs something higher level.